Apache-Arrow-Flight-Tester/main.py

```diff
@@ -90,6 +90,42 @@ def list_actions(flight_client):
     print(list(response))
 
 
+def update_object_store_to_s3(flight_client, endpoint, bucket_name, access_key_id, secret_access_key):
+    object_store_type_bytes = str.encode("s3")
+    object_store_type_size = len(object_store_type_bytes).to_bytes(2, byteorder="big")
+
+    arguments = [endpoint, bucket_name, access_key_id, secret_access_key]
+    action_body = object_store_type_size + object_store_type_bytes + create_update_object_store_action_body(arguments)
+
+    result = flight_client.do_action(pyarrow.flight.Action("UpdateRemoteObjectStore", action_body))
+
+    print(list(result))
+
+
+def update_object_store_to_azure_blob_storage(flight_client, account, access_key, container_name):
```

Comment:
Maybe refactor the shared code in `update_object_store_to_s3()` and `update_object_store_to_azure_blob_storage()` into a separate function as only the object store type and the contents of `arguments` differ.

Reply:
Created one shared function and moved information about what arguments are needed to the doc comment.